content/blog/2021/06/2021-06-17-libera-chat.adoc

```diff
@@ -0,0 +1,96 @@
+---
+:layout: post
+:title: "Jenkins IRC moves to Libera Chat"
+:tags:
+- community
+- announcement
+- chat
+authors:
+- markewaite
+- oleg_nenashev
+links:
+  discourse: https://community.jenkins.io/t/jenkins-irc-moved-to-libera-chat/146
+description: >
+  In this article we announce migration of the Jenkins community from Freenode IRC to Libera Chat.
+  See the blogpost for the migration details and next steps for our community.
+opengraph:
+  image: /images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png
+---
+
+image:/images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png[Jenkins IRC now on Libera Chat, role=center, link="/chat/#internet-relay-chat-irc"]
+
+We are happy to announce that
+the Jenkins community has moved all its link:/chat/#internet-relay-chat-irc[official IRC channels] to link:https://libera.chat/[Libera chat].
+We have started our migration on May 26 as a response to the 
+link:https://www.theregister.com/2021/05/26/freenode_irc_takeover/[hostile takeover]
+of hundreds of open source community channel by the new Freenode management.
+As decided  by the Jenkins governance meeting on June 16th, 
+Libera Chat IRC channels are the only official channels going forward.
+
+A more detailed history of the transition is available on the link:https://groups.google.com/g/jenkinsci-dev/c/pEcun5ZSfgo/m/ztRg03VNAgAJ[Jenkins developers mailing list].
+
+=== What did we move?
+
+We have created the following IRC chats:
+
+that currently live on freenode to libera.chat:
+
+* `#jenkins` - general discussion channel for Jenkins users and developers.
+  We no longer consider it a central chat for all things Jenkins, but we intend to keep it available to users
```

Comment:
This message is confusing as we still mention that channel as the default communication channel
-> https://www.jenkins.io/chat/#jenkins

I would just remove that mention for now until we decide which channel should be the default one

Reply:
Will submit a follow up PR